CONTRIBUTING.md

```diff
@@ -2,35 +2,41 @@
 
 Hi! We're really excited that you are interested in contributing to ACPaaS UI. Before submitting your contribution though, please make sure to take a moment and read through the following guidelines.
 
-- [Code of Conduct](CODE_OF_CONDUCT.md)
-- [Issue Reporting Guidelines](#issue-reporting-guidelines)
-- [Pull Request Guidelines](#pull-request-guidelines)
-- [Development Setup](#development-setup)
-- [Project Structure](#project-structure)
+- [ACPaaS UI Contributing Guide](#acpaas-ui-contributing-guide)
+  - [Issue Reporting Guidelines](#issue-reporting-guidelines)
+  - [Pull Request Guidelines](#pull-request-guidelines)
+  - [Development Setup](#development-setup)
+    - [Committing Changes](#committing-changes)
+    - [Commonly used NPM scripts](#commonly-used-npm-scripts)
+  - [Releases](#releases)
+  - [Credits](#credits)
 
 ## Issue Reporting Guidelines
 
-- Always use [GitHub Issues][github-issues] to create new issues.
+- Use the [#acpaas-ui-react channel][acpaas-ui-react-slack] on slack to ask questions or discuss changes to the code itself.
+- Use [GitHub Issues][github-issues] to report bugs, request features, ask policy questions or propose policy changes.
 
-## Pull Request Guidelines
-
-- The `master` branch is basically just a snapshot of the latest stable release. All development should be done in dedicated branches. **Do not submit PRs against the `master` branch.**
+  > Note: policy questions are about the way ACPaaS UI is developed and released, or about its architecture. We use github issues to have a public archive of these discussions.
 
-- Checkout a topic branch from the relevant branch, e.g. `dev`, and merge back against that branch.
+## Pull Request Guidelines
 
-- Work in the `src` folder and **DO NOT** check in `dist` in the commits.
+- The `master` branch is where we ship releases from. Always send PR's to `master`.
```

Comment:
Small typo (probably in the other repo too): `PRs` (not `PR's`).

Reply:
Al jaren doe ik dat verkeerd blijkbaar. Ik zal erop letten.